rspec-mode.el

```diff
@@ -752,15 +752,21 @@ or a cons (FILE . LINE), to run one example."
           (t (rspec-project-root (file-name-directory (directory-file-name directory)))))))
 
 (defun rspec--include-fg-syntax-methods-p ()
-  "Check whether FactoryGirl::Syntax::Methods is included in spec_helper."
+  "Check whether FactoryGirl::Syntax::Methods is included in rails_helper or spec_helper."
   (cl-case rspec-snippets-fg-syntax
     (full nil)
     (concise t)
     (t
-     (with-temp-buffer
-       (insert-file-contents
-        (concat (rspec-project-root) "spec/spec_helper.rb"))
-       (re-search-forward "include +FactoryGirl::Syntax::Methods" nil t)))))
+     (or
+      (when file-exists-p (expand-file-name "spec/rails_helper.rb" rspec-project-root)
+            (with-temp-buffer
+              (insert-file-contents
+               (expand-file-name "spec/rails_helper.rb" rspec-project-root))
+              (re-search-forward "include +FactoryGirl::Syntax::Methods" nil t)))
+      (with-temp-buffer
+        (insert-file-contents
+         (expand-file-name "spec/spec_helper.rb" rspec-project-root))
+        (re-search-forward "include +FactoryGirl::Syntax::Methods" nil t))))))
```

Comment:
Could you try to avoid duplication here?

Maybe use `cl-find-if` and a list with both filenames.